test/spec/automatic/tagspec.js

```diff
@@ -77,10 +77,232 @@ describe("Cloudinary.ImageTag", function() {
   config = {
     'cloud_name': 'test123'
   };
+
+  let minWidth = 100;
+  let maxWidth = 399;
+  let breakpointList = [minWidth, 200, 300, maxWidth];
+  let commonSrcset = {'breakpoints': breakpointList};
+  let publicId = 'sample';
+  let imageFormat = 'jpg';
+  let fullPublicId = publicId + '.' + imageFormat;
+  let commonTransformation = {'effect': 'sepia'};
+  let commonTransformationStr = 'e_sepia';
+  let commonImageOptions = Object.assign({}, config, {
+    transformation: [commonTransformation],
+  });
+
   beforeEach(function() {
     return cl = new cloudinary.Cloudinary(config);
   });
   DEFAULT_UPLOAD_PATH = `${protocol}//res.cloudinary.com/test123/image/upload/`;
+
+  it("Should create srcset attribute with provided breakpoints", function () {
```

Comment:
maybe replace the word "with" to "from" ? 

we generate `srcset` from the provided breakpoints.